services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -116,6 +117,7 @@ export const DashboardPage = ({ reportType }: Props) => {
   }, [reportsByState]);
 
   const enterSelectedReport = async (report: ReportMetadataShape) => {
+    clearSelectedEntity();
```

Comment:
any time a user enters a report, we need to make sure there is no selectedEntity yet, so this clears it out.